LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
+Copyright (c) Nuxt Community
```

Comment:
I guess we could do:
```suggestion
Copyright (c) Nuxt Community
Copyright (c) Rafał Chłodnicki (@rchl)
Copyright (c) Paul Gascou-Vaillancourt (@paulgv)
```

Reply:
LGTM will apply that; hope it now represents the history^^